spdockit-spo/connect-to-spo.md

```diff
@@ -10,14 +10,24 @@ In order for this feature to work, you will first have to configure SPDocKit dat
 
 Make sure that the user running the application has been granted all [user permissions](#internal/requirements/sharepoint-online-user-permissions-requirements) required to run SPDocKit properly.
 
-You can access the __Connect___ form from two places:
+## Connect:
+
+You can access the __Connect__ form from two places:
 * Go to __Backstage > Actions > SharePoint Online__ button
 * Go to the __Home ribbon__ on either __Farm, Site or Permission Explorer__ and click connect
 
 Once the Connect form opens, SharePoint Online radio button will be preselected. Here is what you need to do:
-1. enter the SharePoint Online Site url.
+1. Enter the SharePoint Online Site url.
 1. If this is your fist time making a connection you will need to specify __Username and Password__. If you have already done this step before, you can choose __Existing Credentials__ radio button and pick them from a dropdown or you can use integrated credentials.
 1. If you wish to connect only to the specified site, click OK and finish.  
 However, if you wish you can discover ALL of the Tennant sites and connect to all of them in one step. Click on __Discover Tenant Sites__ radio button and use a check box to select the sites you wish to connnect to.
 
 You can now explore SharePoint Online sites you've chosen.
+
+## Edit: 
+To edit the existing tenant connection, select the tenant on the __Permission Explorer screen__ and click the __Edit button__ located on the __Manage tab__. By ticking or unticking the checkboxes you can choose which site collections you wish to manage. Also, you can change the credentials you use for connecting to each site collection. 
+Alternatively, you can select one specific site collection. On Permissions Explorer screen -> SharePoint sites node -> choose a site collection and click __Edit__ to manage its credentials. 
+
+## Remove:
+To remove a connected site collection, select it, and on the __Manage tab__ click __Remove__.  
+If you wish to remove entire tenant at once -> select the tenant and again, click the __Remove__ button.
```

Comment:
The first *tenant* here needs and article as well, I'd go with a definite one, but either could work.